File: namsor/csharp/src/OSEG.NamsorExamples/ChineseNameCandidatesExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class ChineseNameCandidatesExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("X-API-KEY", "YOUR_API_KEY");

        try
        {
            var response = new ChineseApi(config).ChineseNameCandidates(
                chineseSurnameLatin: "Hong",
                chineseGivenNameLatin: "Yu"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ChineseApi#ChineseNameCandidates: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: namsor/typescript-axios/src/ChineseNameCandidatesExample.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ChineseApi(configuration).chineseNameCandidates(
  "Hong", // chineseSurnameLatin
  "Yu", // chineseGivenNameLatin
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ChineseApi#chineseNameCandidates:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
